app/models/family.rb

```diff
@@ -2,5 +2,8 @@ class Family < ApplicationRecord
   belongs_to :user
   has_many :personal_info, as: :informable, dependent: :destroy
   accepts_nested_attributes_for :personal_info
-  has_and_belongs_to_many :lic
+  has_and_belongs_to_many :life_insurances
+
+  #validation
+  validates :health, :death_age, :death_year, :death_reason, presence: true
```

Comment:
@VaibhaviJoshi-BTC : What about if person is alive, so is it necessary for the policy holder to enter death age & other related fields?